src/replication/adapter/azurecr/adapter.go

```diff
@@ -0,0 +1,118 @@
+package azurecr
```

Comment:
I think the package name should be `azureacr`?

Reply:
It's `azurecr` in Azure officially.